src/app/application.cpp

```diff
@@ -307,21 +307,21 @@ void Application::runExternalProgram(BitTorrent::TorrentHandle *const torrent) c
 #endif
 }
 
-void Application::sendNotificationEmail(BitTorrent::TorrentHandle *const torrent)
+void Application::sendNotificationEmail(const BitTorrent::TorrentHandle *torrent)
 {
     // Prepare mail content
-    QString content = QObject::tr("Torrent name: %1").arg(torrent->name()) + "\n";
-    content += QObject::tr("Torrent size: %1").arg(Utils::Misc::friendlyUnit(torrent->wantedSize())) + "\n";
-    content += QObject::tr("Save path: %1").arg(torrent->savePath()) + "\n\n";
-    content += QObject::tr("The torrent was downloaded in %1.",
-                  "The torrent was downloaded in 1 hour and 20 seconds")
-            .arg(Utils::Misc::userFriendlyDuration(torrent->activeTime())) + "\n\n\n";
-    content += QObject::tr("Thank you for using qBittorrent.") + "\n";
+    const QString content = QObject::tr("Torrent name: %1").arg(torrent->name()) + "\n"
```

Comment:
Maybe replace `QObject::tr` with `tr` to have proper context in translation files?

Reply:
OK, done.